packages/extension-ui/src/Popup/ImportSeed.test.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+
+import { configure, mount, ReactWrapper } from 'enzyme';
+import {
+  ActionContext,
+  themes,
+  InputWithLabel,
+  Input,
+  Button
+} from '@polkadot/extension-ui/components';
+import { ThemeProvider } from 'styled-components';
+import Import from './ImportSeed';
+import { flushAllPromises } from '../testHelpers';
+import { act } from 'react-dom/test-utils';
+import { BrowserRouter } from 'react-router-dom';
+import DerivationPath, { OptionsLabel } from '@polkadot/extension-ui/partials/DerivationPath';
+import * as messaging from '@polkadot/extension-ui/messaging';
```

Comment:
```suggestion
import { act } from 'react-dom/test-utils';
import { BrowserRouter } from 'react-router-dom';
import { ThemeProvider } from 'styled-components';

import {
  ActionContext,
  themes,
  InputWithLabel,
  Input,
  Button
} from '@polkadot/extension-ui/components';
import DerivationPath, { OptionsLabel } from '@polkadot/extension-ui/partials/DerivationPath';
import * as messaging from '@polkadot/extension-ui/messaging';

import Import from './ImportSeed';
import { flushAllPromises } from '../testHelpers';
```
Small import reformat.